test/hermes_test/commands/harvest/test_cff.py

```diff
@@ -64,3 +64,44 @@ def test_validate_success(valid_minimal_cff):
 
 def test_validate_fail():
     assert harvest._validate(pathlib.Path("foobar"), {}) is False
+
+
+
+# Regression test for https://github.com/hermes-hmc/workflow/issues/112
+
+@pytest.fixture
+def codemeta_with_email():
+    return json.loads("""{
+      "@context": "https://doi.org/10.5063/schema/codemeta-2.0",
+      "@type": "SoftwareSourceCode",
+      "author": [
+        {
+          "@type": "Person",
+          "email": "em@il.notexist",
+          "name": "Author"
+        }
+      ],
+      "name": "Title"
+    }""")
+
+
+@pytest.fixture()
```

Comment:
```suggestion
@pytest.fixture
```

... just for consistency ;)

Reply:
Done in local commit that I forgot to push.